curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/649f0fb16b7e27027526f4da.md

```diff
@@ -0,0 +1,294 @@
+---
+id: 649f0fb16b7e27027526f4da
+title: Step 16
+challengeType: 0
+dashedName: step-16
+---
+
+# --description--
+
+Add an `if` statement where if `rolls` equates to `3`, it will send a `window.alert` message saying `You have made three rolls this round. Please select a score.`
+
+# --hints--
+
+You should add an `if` statement where the condition is `rolls` equates to `3`. Use the strict equality ( `===` ) operator.
+
+```js
+assert.match(code, /if\s*\(\s*rolls\s*===\s*3\s*\)\s*{/);
+```
+
+After the condition is met, you should add a `window.alert` message saying `You have made three rolls this round. Please select a score.`
```

Comment:
This should be two tests. One to check that `window.alert` is called, and the second to check the arugment passed is correct.